app/components/windows/Main.vue.ts

```diff
@@ -168,27 +170,34 @@ export default class Main extends Vue {
 
   hasLiveDock = true;
 
+  windowResizeTimeout: number;
+
   windowSizeHandler() {
+    this.onResizeStartHandler();
```

Comment:
As far as I can tell, this is going to cause us to commit a new mutation in the windows service every time a resize event happens, which is many times per second while resizing.  This mutation will be synced across all windows through the main process, which is pretty slow.  If that is happening (I think it likely is) then we should make sure we only commit 1 mutation at the start of resizing and one at the end.  This will be much better for smooth resizing on slow machines I think.

It's also going to be doing a synchronous IPC to the windows service on every resize event (`updateStyleBlockers`) which is also going to slow things down considerably.